pep/app/pods/components/document/text/template.hbs

```diff
@@ -1,5 +1,11 @@
 <div
-    class="document-text"
+    class="document-text
+
+
+        {{if this.currentUser.preferences.glossaryFormattingEnabled "glossary-highlight"}}
+
+
+        {{this.currentUser.textJustification.class}}"
```

Comment:
i'd say its 95% likely they'll want the text justification setting to apply to anywhere we display abstracts too (document preview pane, expert pick of the day, etc), does this apply to those instances?

also why all the extra whitespace here?

Reply:
We don't use the document text component in the document preview pane (I think we probably should, but haven't gotten to looking at it yet). It applies everywhere else though.

The whitespace is from my auto formatter. Every time I fix it, it gets added back in

Reply:
thats weird, i can see adding a single linebreak/space between them, by why two. 🤔

and ok, in that case maybe we can just apply the `{{{{this.currentUser.textJustification.class}}` class directly to the preview pane component's abstract content container element?